# 커스텀 메시지: SELD 이벤트에 타임스탬프 붙여서 퍼블리시
std_msgs/Header header
float64 time
uint8[]   l_r